presto-docs/src/main/sphinx/release/release-0.275.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.275
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator to correctly identify running queries as leaked.
+* Improve the memory limit error message with allocationTag.
+* Add :func:`trimmed_mean` for :ref:`tdigest <tdigest_type>`. The prototype of the function is: trimmed_mean(tdigest: TDIGEST, lower_quantile: DOUBLE, upper_quantile: DOUBLE) -> DOUBLE.
```

Comment:
Not quite getting the meaning of the second sentence: "The prototype of the function is: trimmed_mean(tdigest: TDIGEST, lower_quantile: DOUBLE, upper_quantile: DOUBLE) -> DOUBLE.". Usually we just say "Add function ..."